packages/frontend/src/pages/instructlab/StartInstructLabContainer.svelte

```diff
@@ -0,0 +1,141 @@
+import TrackedTasks from '/@/lib/progress/TrackedTasks.svelte';
+import { instructlabClient, studioClient } from '/@/utils/client';
+import type { Task } from '@shared/src/models/ITask';
+import { onMount } from 'svelte';
+
+interface Props {
+  // The tracking id is a unique identifier provided by the
+  // backend when calling requestCreateInstructLabContainer
+  trackingId?: string;
+}
+
+let { trackingId }: Props = $props();
+
+// The container provider connection to use
+let containerProviderConnection: ContainerProviderConnectionInfo | undefined = $state(undefined);
+
+// Filtered connections (started)
+let startedContainerProviderConnectionInfo: ContainerProviderConnectionInfo[] = $derived(
+  $containerProviderConnections.filter(connection => connection.status === 'started'),
+);
+
+// If the creation of the InstructLab fail
+let errorMsg: string | undefined = $state(undefined);
+// The containerId will be included in the tasks when the creation
+// process will be completed
+let containerId: string | undefined = $state(undefined);
+// available means the container is started
+let available: boolean = $derived(!!containerId);
+// loading state
+let loading = $derived(trackingId !== undefined && !errorMsg);
+
+onMount(async () => {
+  containerId = await instructlabClient.getInstructlabContainerId();
+});
+
+$effect(() => {
+  // Select default connection
+  if (!containerProviderConnection && startedContainerProviderConnectionInfo.length > 0) {
+    containerProviderConnection = startedContainerProviderConnectionInfo[0];
+  }
+});
+
+function processTasks(trackedTasks: Task[]): void {
+  // Check for errors
+  // hint: we do not need to display them as the TasksProgress component will
+  errorMsg = trackedTasks.find(task => task.error)?.error;
+
+  const task: Task | undefined = trackedTasks.find(task => 'containerId' in (task.labels ?? {}));
+  if (task === undefined) return;
+
+  containerId = task.labels?.['containerId'];
+}
+
+// Submit method when the form is valid
+async function submit(): Promise<void> {
+  errorMsg = undefined;
+  try {
+    trackingId = await instructlabClient.requestCreateInstructlabContainer({
+      connection: $state.snapshot(containerProviderConnection),
+    });
+    router.location.query.set('trackingId', trackingId);
+  } catch (err: unknown) {
+    console.error('Something wrong while trying to create the InstructLab container.', err);
+    errorMsg = String(err);
+  }
+}
+
+// Navigate to the new created service
+function openInstructLabContainer(): void {
+  instructlabClient.routeToInstructLabContainerTerminal(containerId!).catch(console.error);
+}
+
+function openDocumentation(): void {
+  studioClient.openURL('https://docs.instructlab.ai').catch(console.error);
+}
+</script>
+
+<FormPage title="Run InstructLab as a container">
+  <svelte:fragment slot="content">
+    <div class="flex flex-col w-full">
+      <header>
+        <div class="w-full flex flex-row px-8">
```

Comment:
```suggestion
      <header class="mx-5 mt-5">
        <div class="w-full flex flex-row">
```

fix padding

![image](https://github.com/user-attachments/assets/c11415ba-49f1-4d7b-bb07-2b4dd3cd4312)